### DISCUSSION TITLE
nwg-panel: configure hover color and workspace icon

### DISCUSSION THREAD

**Discussion by paladhammika** [nwg-piotr/nwg-shell#127] [Q&A]
Thanks for the great project. It met my need for having a simple, all inclusive (tray, notifications, calendar) status bar without having to go through the headache of configuring something like eww.

When placing the cursor over the workspaces, clock, or tray, there appears a thin green border around the area, how can I change this in the css?

Also, would it be possible to change the workspace from a number to an icon? Like below in polybar.

![image](https://github.com/nwg-piotr/nwg-shell/assets/93992898/f75dbee6-1b23-4029-a5c7-6e56147a3770)

**Comment by nwg-piotr** [comment]
> there appears a thin green border around the area, how can I change this in the css?

This is done by setting [state flags](https://docs.gtk.org/gtk4/flags.StateFlags.html), to indicate clickable elements. I have no idea if it can be changed in css, but it will look different in various GTK themes (border / background / nothing at all). E.g. in the Nordic theme I use it's a background:

![image](https://github.com/nwg-piotr/nwg-shell/assets/20579136/32d81a0f-cc67-4467-924b-cc2472e456c8)

> would it be possible to change the workspace from a number to an icon? 

Depends on which module you mean. E.g. if you ` hyprctl dispatch renameworkspace 4 Web`:

- in the Workspaces module it will only append "Web" to the workspace number;
- in the Taskbar module, "Web" will appear instead of "4".
